// Copyright (c) 2011 by  Marco TÃºlio Ribeiro <marcotcr@gmail.com>,
//                        Tiago Alves Macambira <tmacam@burocrata.org>
// All rights reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.package rendero;

message AggregatorValue {
 optional bool bool_value = 1;
 optional double double_value = 2;
 optional int64 int64_value = 3;

 // optionally AggregatorValues can use extensions
 extensions 10 to max;
}


message AggregatorGroup {
  repeated AggregatorValue aggregators = 1;
};

message AggregatorPartials {
  required string name = 1;
  required AggregatorGroup values = 2;
}
